skimage/feature/texture.py

```diff
@@ -291,3 +293,130 @@ def local_binary_pattern(image, P, R, method='default'):
     image = np.ascontiguousarray(image, dtype=np.double)
     output = _local_binary_pattern(image, P, R, methods[method.lower()])
     return output
+
+def multiblock_local_binary_pattern(int_image, x, y, width, height):
```

Comment:
I brought this up because, `20000x2049` is a huge number, and those many Python calls will certainly slow things down a lot. Can you benchmark a single call on a `20x20` window ? It looks more like this will need a Cython equivalent. And if we have both Python/Cython code, it will be redundant. I was thinking we could have a Cython version now itself and a thin Python wrapper around it.